app/models/grda_warehouse/census/project_batch.rb

```diff
@@ -40,7 +40,9 @@ def build_census_batch
                 (inventory.InventoryEndDate.blank?)) ||
             ((inventory.InformationDate.blank? && inventory.InventoryStartDate.present? && inventory.InventoryStartDate < date) &&
                 (inventory.InventoryEndDate.present? && inventory.InventoryEndDate > date))
-          end.sum(&:beds) rescue 0
+          end
+          # preserve behavior of returning 0 if any bed values are nil (rather than compact.sum)
+          census_item.beds = filtered_beds.all?(&:present?) ? filtered_beds.sum : 0
```

Comment:
@eanders - I'm guessing that "beds" is sometimes nil here which is triggering the deprecation warning in sentry. I assume the rescue was there to recover from trying to add nil values. I replaced it with a conditional which seems equivalent.

Reply:
Thank you!